x/gov/simulation/msgs.go

```diff
@@ -103,16 +101,16 @@ func simulateHandleMsgSubmitProposal(msg gov.MsgSubmitProposal, handler sdk.Hand
 	return ok
 }
 
-func simulationCreateMsgSubmitProposal(r *rand.Rand, sender simulation.Account) (msg gov.MsgSubmitProposal, err error) {
-	deposit := randomDeposit(r)
-	msg = gov.NewMsgSubmitProposal(
-		gov.NewTextProposal(
-			simulation.RandStringOfLength(r, 5),
-			simulation.RandStringOfLength(r, 5),
-		),
-		deposit,
-		sender.Address,
+// SimulateTextProposalContent returns random text proposal content.
+func SimulateTextProposalContent(r *rand.Rand) gov.Content {
+	return gov.NewTextProposal(
+		simulation.RandStringOfLength(r, 5),
+		simulation.RandStringOfLength(r, 5),
```

Comment:
Maybe simulate a few with too-long content (is the length checked)?

Reply:
It is length checked; I'll increase it 👍

Reply:
It has to be within bounds though, otherwise `ValidateBasic` will fail and simulation as a result will fail.

Reply:
:+1: